runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerPlugin.java

```diff
@@ -233,6 +244,28 @@ private int checkContainerForPrayer(Item[] items)
 		return total;
 	}
 
+	float getTickProgress()
+	{
+		long timeSinceLastTick = Duration.between(startOfLastTick, Instant.now()).toMillis();
+
+		float tickProgress = (timeSinceLastTick % 600) / 600f;
+		tickProgress = Math.min(tickProgress, 1);
+		return tickProgress;
```

Comment:
I would return tickProgress * Math.PI as that is used in both plugins

Reply:
That is used in a double.

Reply:
then return double :D